src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/MonthCalendarAccessibleObjectTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Windows.Forms.Tests.AccessibleObjects
+{
+    public class MonthCalendarAccessibleObjectTests : IClassFixture<ThreadExceptionFixture>
+    {
+        [WinFormsFact]
+        public void MonthCalendarAccessibleObject_CalendarCountIsCorrect_IfCalendarContainerIsBig ()
+        {
+            var calendar = new MonthCalendar();
```

Comment:
This probably needs to be disposed?
```suggestion
            using var calendar = new MonthCalendar();
```

Reply:
https://github.com/dotnet/winforms/blob/master/Documentation/testing.md#dispose-created-objects